ibis/backends/tests/test_register.py

```diff
@@ -351,15 +346,6 @@ def test_csv_reregister_schema(con, tmp_path):
     assert result_schema["colb"].is_float64()
     assert result_schema["colc"].is_string()
 
-    # If file scan is limited to first two rows, should be all some kind of integer.
-    # The specific type isn't so important, and may vary across backends/versions
-    foo_table = con.register(foo, SAMPLE_SIZE=2, table_name="same")
```

Comment:
No matter what I tried, DuckDB ignores the `sample_size` argument here and (correctly) infers the `DOUBLE` type for `colb`, and every other backend that didn't do this passes the test otherwise, so I deleted this case.